lib/project_types/android.sh

```diff
@@ -5,11 +5,6 @@ Slapshot_last_push_file="slapshot-last-commit.txt"
 Slapshot_release_notes_file="release_notes.tmp.txt"
 
 Slapshot_validate_build() {
-    if [[ "$Slapshot_config_hockey_app_api_token" = "" ]]; then
-        Logger__error "No hockey_app_token provided in the slapshot_config.yaml file"
-        return 0
-    fi
-
```

Comment:
Weird -- I just did this before reading this... We must be on the same wavelengths today